be/src/services/entity.service.ts

```diff
@@ -130,10 +130,11 @@ export class EntityService {
     const { options, filter } = this.transformToMongoDBFilterOption(query, id);
     // @ts-ignore
     const dbos: Entity[] = (await EntityService.collection().find(filter, options).toArray()) as Entity[];
+    const size = dbos.length
     return {
       offset: query.offset,
       rows: dbos.length,
-      totalItems: 0, // TODO
+      totalItems: size, // TODO
```

Comment:
1. Remove the `TODO` comment
2. Using a variable with the same content as the above row is false, then we would not net totalItems at all. Instead:
3. We need the information of how many items are in total without any paging - no consideration of offset nor rows - are found for the filter with options on the DB. Is that correct @ja-fra ?